src/library/treeitem.cpp

```diff
@@ -82,34 +82,22 @@ void TreeItem::initFeatureRecursively(LibraryFeature* pFeature) {
     }
 }
 
-TreeItem* TreeItem::appendChild(
-        std::unique_ptr<TreeItem> pChild) {
-    insertChild(m_children.size(), pChild.get()); // transfer ownership
-    return pChild.release();
-}
-
 TreeItem* TreeItem::appendChild(
         QString label,
         QVariant data) {
-    auto pNewChild = std::make_unique<TreeItem>(
+    std::unique_ptr<TreeItem> pNewChild = std::make_unique<TreeItem>(
```

Comment:
this feels redundant. I thought declaration with `auto` was fine since the return type is easily inferred from `make_unique`.

Reply:
Done.